examples/many-pages/README.md

```diff
@@ -1,6 +1,6 @@
 # Info
 
-This example illustrates webpack's algorthim for automatic deduplication using `optimization.splitChunks`.
```

Comment:
Edit `template.md` is the same way. README.md is a generated file.

Reply:
Sorry, I can't understand. What do you mean by 'Edit `template.md` is the same way'? Should I edit `template.md`?

Reply:
@erolaliyev I guess good to close this PR since this typo was fixed in https://github.com/webpack/webpack/pull/7693.